### ISSUE TITLE
Can we setup an internal validator?

### ISSUE THREAD

**Issue by luisfcorreia** [qualweb/qualweb#175]
Hi,

this is more of a question and not a true issue.

We're trying to setup an internal WCAG validation solution that should not require Internet access to perform all needed validation.

This is a requirement for accessibility tests on internal, non production services that cannot be "exposed" to the Internet.

I've yet to find where we can have the code for the validator endpoint referenced in README.md, "validator": "http://127.0.0.1/validate"

(I have very little JS/TS expertese)

Thanks in advance

**Comment by luisfcorreia** [comment]
Well, 

maybe I can give further insight of what our plan is.

We run some automated tests using Java and Selenium WebDriver.
During these tests, we want to validate some parts of those pages, in which we can get full HTML source from Selenium. The current plan is to then "send this HTML" for internal validation, as many times as we want during this test run and then gather reports from each page.

As of now, we're unsure on how to actually do it from within Java but I personally have time available to figure out more.

**Comment by luisfcorreia** [comment]
No further help is needed, we've figured it out.

Thanks for clarifying no external validation is needed!
